parlai/utils/distributed.py

```diff
@@ -356,9 +356,11 @@ def slurm_distributed_context(opt):
     except subprocess.CalledProcessError as e:
         # scontrol failed
         raise e
-    except FileNotFoundError:
+    except FileNotFoundError as e:
         # Slurm is not installed
-        raise RuntimeError('SLURM does not appear to be installed.')
+        raise RuntimeError(
```

Comment:
hm maybe we should only try-catch the `subprocess.check_output` section. That way we won't eat the FileNotFoundError from way down in the stack

Reply:
Don't have enough context to know what that means exactly... seeing as how it's not clear if anyone else has run into this, maybe I'll land this as is for now?